addons/stock_fleet/models/fleet_vehicle_model.py

```diff
@@ -29,3 +29,14 @@ def _compute_weight_capacity_uom_name(self):
 
     def _compute_volume_capacity_uom_name(self):
         self.volume_capacity_uom_name = self.env['product.template']._get_volume_uom_name_from_ir_config_parameter()
+
+
+class FleetVehicle(models.Model):
+    _inherit = 'fleet.vehicle'
+
+    def _compute_display_name(self):
+        if self.env.context.get('from_picking_batch'):
```

Comment:
Not needed. Remove the context condition here, so that anyone can access this value later.

Reply:
Okay, I see. Actually, I'm not sure if it's necessary at all or not to add the acquisition date on the `name` field instead of `display_name`. I'll consult the vidange team.